web/utils/color.ts

```diff
@@ -0,0 +1,48 @@
+export const getRainbowColor = (
+  numberOfSteps: number,
+  step: number
+): string => {
+  let red, green, blue;
+  let h = step / numberOfSteps;
+  let i = ~~(h * 6);
```

Comment:
Hva er det her for noen insane syntaks? 👀 

Her tror jeg at jeg hadde foretrukket en litt mer leselig approach 😬  Jeg ser for meg at HSL-formatet kanskje kan være enklere å bruke her. Har aldri brukt det før, men det er visst ganske fint støttet. Og da tror jeg kanskje du kan forenkle dette til noe ala det her? 🤔 

```tsx
export const getRainbowColor = (numberOfSteps: number, step: number) => {
   // hue er hvor mange grader man skal bevege seg på fargesirkelen
  const hue = Math.floor(360 * step / numberOfSteps);
  
  // Hvor saturert (fargefull) fargen er. 100 % er maks farge, 0 % er grå
  const saturation = "100%";

  // Hvor lys fargen er 100 % er hvit, 0 % er svart, 45 % er litt mørkere enn maks farge.
  const brightness = "45%";

  return `hsl(${hue}, ${saturation}, ${brightness});`;
}
```

Reply:
Eller en hakket mer kompakt versjon:

```tsx
export const getRainbowColor = (numberOfSteps: number, step: number) => {
  const hue = Math.floor(360 * step / numberOfSteps);
  return `hsl(${hue}, 100%, 45%);`;
};

Reply:
Jeg er ganske stolt av meg selv.

Reply:
Hahah, det ser mye bedre ut ja helt enig 👍

Var veldig inspirert av en løsning jeg fant på nettet, men denne var jo genial! 💡

Reply:
Jeg har ikke sjekket at den funker altså, men burde være relativt close

Reply:
Det funka kjempe bra 💯